apps/project/schema.py

```diff
@@ -295,6 +297,26 @@ def resolve_regions(root, info, **kwargs):
         return info.context.dl.project.public_geo_region.load(root.pk)
 
 
+class RecentActivityType(graphene.ObjectType):
+    id = graphene.ID(required=True)
+    created_at = graphene.DateTime()
+    project = graphene.Field(ProjectType)
+    created_by = graphene.Field(UserType)
+    type = graphene.Field(RecentActivityTypeEnum, required=True)
+    type_display = EnumDescription(required=True)
+
+    def resolve_created_by(root, info, **kwargs):
+        id = int(root['created_by'])
+        return info.context.dl.project.users.load(id)
+
+    def resolve_project(root, info, **kwargs):
+        id = int(root['project'])
+        return info.context.dl.project.projects.load(id)
+
+    def resolve_type_display(root, info, **kwargs):
+        return root['type']
```

Comment:
Can you confirm this provides label instead of value?

Reply:
@thenav56 , here type is not generated from Choice fieds insead it is generate as models.Value('entry-comment', output_field=models.CharField()), so it return text string.